meta-mender-raspberrypi-demo/README.md

```diff
@@ -0,0 +1,30 @@
+# meta-mender-raspberrypi-demo
+This README file contains information on the contents of the /meta-mender/meta-raspberrypi-demo layer.
+
+Please see the corresponding sections below for details.
+
+## Dependencies
+============
+
+  URI: git://git.yoctoproject.org/meta-raspberrypi
+  branch: rocko
+  revision: HEAD:20358ec57a8744b0a02da77b283620fb718b0ee0
+
```

Comment:
Generally I try to avoid adding revisions as they will be out of date quickly (probably are already.). I would say branch is enough.